app/src/components/v-upload.vue

```diff
@@ -80,13 +81,22 @@ function useUpload() {
 
 				uploadedFiles && emit('input', uploadedFiles);
 			} else {
+				const version = props.fileId
+					? {
+							version: (props?.fileVersion || 0) + 1,
+					  }
+					: {};
+
 				const uploadedFile = await uploadFile(Array.from(files)[0] as File, {
 					onProgressChange: (percentage) => {
 						progress.value = percentage;
 						done.value = percentage === 100 ? 1 : 0;
 					},
 					fileId: props.fileId,
-					preset,
+					preset: {
+						...preset,
+						...version,
+					},
```

Comment:
While this solution works for app-users, it doesn't account for users replacing files directly in the API. I think this version number has to be read-incremented-set on the API side rather than the app side, which due to the app using the API will cover both entry points for users 🙂

Reply:
We already have some logic in place around existing versus new files in the file upload handling on the `FilesService`, so if I'm not mistaken we should be able to add in the `version + 1` handling here: 

https://github.com/directus/directus/blob/a42633b657dde33912d11c30b51f162139585124/api/src/services/files.ts#L50-L65

Reply:
I will update the implementation.

Does the `version` key still make sense, or is there a better nomenclature that makes sense here? Such as `replaced_version`, `modified_version`, or another option?

Reply:
Good question! This would both apply to replacements and edits made in the image editor in the app (seeing those use the "replace" endpoint), so `version` works for me. 

Alternatively a `modified_[something]` would also make sense, but conflicts with the `modified_on` flag, which to your earlier point updates all the time, not just for file changes 🤔

Reply:
So lets go with `version` 👍🏻